test/unit/JsonService.test.ts

```diff
@@ -148,5 +182,314 @@ describe("JsonService", () => {
             // assert
             expect(result).toEqual(json);
         });
+
+        it("should work with custom jwtHandler", async () => {
+            // arrange
+            const jwtHandler = jest.fn();
+            subject = new JsonService([], jwtHandler);
+            const text = "text";
+            fetchMock.mockResolvedValue({
+                status: 200,
+                headers: new Headers({
+                    "Content-Type": "application/jwt"
+                }),
+                text: () => Promise.resolve(text)
+            });
+
+            // act
+            await subject.getJson("http://test");
+
+            // assert
+            expect(jwtHandler).toBeCalledWith(text);
+        });
+    });
+
+    describe("postForm", () => {
+        it("should return a promise", async () => {
+            // act
+            const p = subject.postForm("http://test", { "a": "b" });
+
+            // assert
+            expect(p).toBeInstanceOf(Promise);
+            // eslint-disable-next-line no-empty
+            try { await p; } catch {}
+        });
```

Comment:
do we need tests like this? They don't add to the line coverage and TypeScript already enforces type safety.